src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -42,43 +42,88 @@ const StatusMenuItem = () => {
         padding-right: 15px;
       `}
     >
-      Register Samples
-      {/* {isSubmissionSystemDisabled ? (
-        <Icon name="lock" fill="accent3_dark" width="15px" />
-      ) : clinicalRegistrationHasError ? (
-        <Icon name="exclamation" fill="error" width="15px" />
-      ) : clinicalRegistrationInProgress ? (
-        <Icon name="ellipses" fill="warning" width="15px" />
-      ) : null} */}
+      {children}
     </div>
   );
 };
 
-export default function ProgramMenu({ searchQuery }: { searchQuery: string }) {
-  const { data, loading, error } = useQuery(SIDEMENU_PROGRAMS, {
+const parseGQLResp = (data: SideMenuQuery) => {
+  const clinicalErrors = data.clinicalData.clinicalErrors;
+  const clinicalDataHasErrors =
+    (clinicalErrors && clinicalErrors.length > 0) || false;
+
+  const clinicalRegistration = data && data.clinicalRegistration;
```

Comment:
Could just use a `?` I think `data?.clinicalRegistration`
Also if `data` is potentially falsey then I think clinicalErrors would need `?` guards as well